pubspec.yaml

```diff
@@ -14,7 +14,7 @@ executables:
   moronepo:
 
 dependencies:
-  args: ^1.4.0
+  args: <=1.5.2
```

Comment:
Why not `1.6.0`?

Reply:
I got an error when running on CI that said that flutter dependency used 1.5.2 :/

Reply:
Couldn't find a way to make the build pass on CI. Less than 1.6.0 works though (1.5.4).